src/components/Tooltip/Tooltip.tsx

```diff
@@ -56,113 +59,263 @@ const paddingsBottom = {
     large: "tw-pb-4",
 };
 
-const TooltipComponent: ForwardRefRenderFunction<HTMLDivElement, TooltipProps> = (
-    {
-        content,
-        tooltipIcon,
-        heading,
-        headingIcon,
-        linkUrl,
-        linkLabel,
-        brightHeader,
-        buttons,
-        tooltipAriaProps = {},
-        style,
-        children,
-        popperAttributes,
-    },
-    tooltipContainerRef,
-) => {
+export enum TooltipPosition {
+    Top = "top",
+    Right = "right",
+    Bottom = "bottom",
+    Left = "left",
+}
+
+export enum TooltipAlignment {
+    Start = "Start",
+    Middle = "Middle",
+    End = "End",
+}
+
+const placementMap: Record<string, VariationPlacement> = {
+    ["top-Start"]: "top-start",
+    ["top-End"]: "top-end",
+    ["bottom-Start"]: "bottom-start",
+    ["bottom-End"]: "bottom-end",
+    ["left-Start"]: "left-start",
+    ["left-End"]: "left-end",
+    ["right-Start"]: "right-start",
+    ["right-End"]: "right-end",
+};
```

Comment:
```suggestion
const placementMap: Record<`${TooltipPosition}-${TooltipAlignment}`, VariationPlacement> = {
    ["top-Start"]: "top-start",
    ["top-End"]: "top-end",
    ["bottom-Start"]: "bottom-start",
    ["bottom-End"]: "bottom-end",
    ["left-Start"]: "left-start",
    ["left-End"]: "left-end",
    ["right-Start"]: "right-start",
    ["right-End"]: "right-end",
};
```
And seems we have different casing